libs/components/forms/src/lib/modules/form-error/form-errors.component.html

```diff
@@ -1,56 +1,58 @@
-<ng-container *ngIf="labelText && showErrors && errors">
-  <sky-form-error
-    *ngIf="errors['required']"
-    errorName="required"
-    [errorText]="'skyux_form_error_required' | skyLibResources: labelText"
-  />
+<ng-container *ngIf="labelText && showErrors">
+  <ng-container *ngIf="errors">
```

Comment:
What was the purpose of this change? It appears to just be adding another `ng-container` that just moves one of the conditions from the old `ngIf`. Am I missing something on this file changing?

Reply:
So the reasoning behind this was it expects a ngControl `ValidationErrors` list or null. It only displays when there are validation errors, and it then displays known error messages. Since file attachment isn't a control value accessor it doesn't have ValidationErrors. Fortunately it doesn't have any of the known errors, but not having a list to pass would mean that _none_ of the form errors show up, not even the custom ones.

The options were to either pass it an empty array or make this change, where form errors can still be shown without being part of the validation errors list. This felt cleaner to me than a hardcoded empty list in file drop.

Reply:
Oh. I see the line that I missed where the `ng-content` is outside of the new `ng-container`. Should we have a unit test for this scenario - to ensure a future engineer doesn't miss that nuance when working here?

Reply:
yeah good idea!